uint8 mode

uint8 READY = 0
uint8 END = 10
uint8 ROUNDABOUT_SEARCH = 1
uint8 INDUCE_ATTACK = 2
uint8 ESCAPE = 3
uint8 SEARCH_ANY = 4
uint8 RED_PURSUIT = 5
uint8 BREAKOUT = 6
